src/components/VersionSelector/types.ts

```diff
@@ -0,0 +1,24 @@
+export interface ActiveVersionData {
+  resource_version: string;
+  api_version: string;
+}
+export interface VersionSelectorProps {
+  resource_versions: string[];
+  active: ActiveVersionData;
+  root_url: string;
+  description?: string;
+  variant?: 'dark' | 'light'; // Not fully implemented
```

Comment:
Are we planning to support these two props? Was just curious

Reply:
Good question. It is available within Redoc and in LG, but I don't believe we ever take advantage of that in our specs. Best to leave it out.